### PR TITLE
Feature/modal profile notifications

### PR DESCRIPTION
**Pull request by yepzdk**
https://platform.dandigbib.org/issues/5388

Add the possibility of showing notification in profile modal.

Screenshot 2022-07-27 at 10 59 02

### CODE REVIEW

File: src/stories/Library/Modals/modal-profile/ModalProfile.stories.tsx [hunk 1]
```diff
@@ -24,6 +24,31 @@ export default {
       control: { type: "text" },
       defaultValue: "Se din profil",
     },
+    notifications: {
+      control: { type: "object" },
+      defaultValue: [
```
Comment (Adamik10) [inline src/stories/Library/Modals/modal-profile/ModalProfile.stories.tsx:29]: Lovely that you adopted this. I'm really enjoying argTypes and default values too in my stories 😼

File: src/stories/Library/Modals/modal-profile/modal-profile.scss [hunk 2]
```diff
@@ -34,3 +63,7 @@
     }
   }
 }
+
+.test-color {
```
Comment (Adamik10) [inline src/stories/Library/Modals/modal-profile/modal-profile.scss:67]: Should this stay here?

**Review by Adamik10 — APPROVED** [review]
Lovely work. Posted one question in the scss file, but it's approved. 👍

File: src/stories/Library/Modals/modal-profile/modal-profile.scss (hunk 2 above)
Comment (yepzdk) [reply src/stories/Library/Modals/modal-profile/modal-profile.scss]: Absolutely not.. Thanks for pointing it out.

**Pull request merged**